src/api/controller/api/publishedDataset.js

```diff
@@ -121,6 +142,7 @@ app.use(async (ctx, next) => {
 
     await next();
 });
+app.use(route.post('/create_resource', createResource));
```

Comment:
I think it's time to introduce a `resource` with correct REST usage. If I remember correctly, we have the PUT/PATCH inside publishedDataset.